misc/iitthesis.cls

```diff
@@ -170,12 +170,42 @@
 \setmainlanguage{english} % Palestine was under British rule after all
 \IfFileExists{bidi.sty}{}{\ClassError{\filename}{The bidi package is not available for loading}{}}
 \AtEndPreamble{
- \setotherlanguage[numerals=arabic,calendar=gregorian]{hebrew}
+ \setotherlanguage[numerals=arabic,calendar=gregorian,fullyear=true]{hebrew}
  \newfontfamily\hebrewfont[Script=Hebrew]{David CLM}
  % These next two lines fix an issue with equation numbering parentheses;
  % see: https://tex.stackexchange.com/a/141437/5640
  \def\maketag@@@#1{\hbox{\m@th\normalfont\LRE{#1}}}
  \def\tagform@#1{\maketag@@@{(\ignorespaces#1\unskip)}}
+ \RequirePackage{hebrewcal}
+
+ % override hebrewcal.tex macros to remove the day of the month:
+ % See: https://tex.stackexchange.com/questions/610388
+ \def\@FormatForHebrew#1#2#3{%
+  \HebrewMonthName{#2}{#3}~%
+  \HebrewYearName{#3}}
+ \def\@FormatForEnglish#1#2#3{%
+  \HebrewMonthNameInEnglish{#2}{#3}~%
+  \number#3}
+
+ % Create datetime2 styles without the day of the month for the Hebrew
```

Comment:
If we use these, do we also need the hebrewcal-related changes?

And, actually - do we need both packages for obtaining the current month and year?

Reply:
I couldn't get `datetime2` working correctly for Hebrew dates. I can try again if you think it is important. Perhaps it is possible to print the Gregorian month and year without `datetime2`, but I thought its approach with the custom styles was more robust.